warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/job/BulkIngestMapFileLoader.java

```diff
@@ -433,9 +434,16 @@ public void run() {
         int fsAccessFailures = 0;
         Path[] jobDirectories = new Path[0];
         int nextJobIndex = 0;
+        boolean firstRun = true;
+        
         try {
             while (true) {
                 try {
+                    if (firstRun) {
+                        
+                        cleanJobDirectoriesOnStartup();
```

Comment:
Why not put this before the while loop and avoid the firstRun flag altogether?

Reply:
Why not, indeed.